.travis.yml

```diff
@@ -0,0 +1,3 @@
+language: dart
+dart:
+  - dev
```

Comment:
testing on stable is good, too.

Also: I encourage folks to include at least the current stable branch explicitly here.

...keeps us honest about actually supporting previous SDKS https://github.com/dart-lang/shelf/blob/master/.travis.yml